tools/react-app-template/webpack.config.js

```diff
@@ -1,20 +1,20 @@
-// (C) 2007-2021 GoodData Corporation
+// (C) 2007-2022 GoodData Corporation
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const CaseSensitivePathsPlugin = require("case-sensitive-paths-webpack-plugin");
 const { DefinePlugin, EnvironmentPlugin, ProvidePlugin } = require("webpack");
 const path = require("path");
 const { URL } = require("url");
-const Dotenv = require("dotenv-webpack");
 require("dotenv").config();
 
 const PORT = 3001;
-const BACKEND_URL = "https://live-examples-proxy.herokuapp.com";
+const BACKEND_URL = "https://public-examples.gooddata.com";
 
 module.exports = (_env, argv) => {
     const isProduction = argv.mode === "production";
     const protocol = new URL(BACKEND_URL).protocol;
-    const proxy = {
-        "/gdc": {
+    const proxy = [
+        {
+            context: ["/api", "/gdc"],
```

Comment:
/gdc is just for bear, do we want to keep it by default?

Reply:
With this notation, proxy will run for both `/api` and `/gdc` paths. Which means users would not have to update this manually if they want to connect to Bear backend. One less step in the guide. I could add a comment in code if you feel like the way it's now may create confusion.